runtime/tools/python/ttrt/common/run.py

```diff
@@ -119,44 +119,44 @@ def initialize_api():
         Run.register_arg(
             name="--enable-async-ttnn",
             type=bool,
-            default=False,
+            default=True,
```

Comment:
This should be defaulted to false as we want to run tests through ttrt without async ttnn enabled which aligns with what runtime is doing.

Reply:
oops forgot to change that back, thanks for the catch

Reply:
fixed